redaxo/src/addons/be_style/plugins/customizer/boot.php

```diff
@@ -22,9 +22,16 @@
         $filenames[] = $this->getAssetsUrl('vendor/codemirror/addon/fold/foldgutter.css');
     }
 
+    $content = '';
     foreach ($filenames as $filename) {
-        echo '/* ' . $filename . ' */' . "\n" . rex_file::get($filename) . "\n";
+        $content .= '/* ' . $filename . ' */' . "\n" . rex_file::get($filename) . "\n";
     }
+
+    header('Pragma: cache');
+    header('Cache-Control: public');
+    header('Expires: ' . date('D, j M Y', strtotime('+1 week')) . ' 00:00:00 GMT');
+    echo $content;
```

Comment:
geht hier auch `rex_response::sendContent(..)`?

Reply:
Hi Markus, wie oben geschroben habe ich das mit sendContent versucht aber nicht hinbekommen.
Mit der jetzigen Lösung zeigen mir die DevTools loading from disk cache ...

Reply:
Hatte folgendes versucht aber es landete nichts im Cache

$lastModified = gmdate('r', filemtime($this->getAssetsUrl('vendor/codemirror/codemirror-compressed.js')));
$etag = md5($content);
rex_response::sendContent($content, 'application/javascript', $lastModified, $etag);

Versuche mit/ohne Buster + eigene Header pragma+cache-control ....

Reply:
du darfst keinen etag/last-modified haben, wenn du willst das der browser cached.

Reply:
Ich habe das noch einmal versucht
rex_response::sendContent($content, 'application/javascript');
rex_response::sendPage($content);

bekomme hier dann einen 304 not modified! Auch mit den folgenden zusätzlichen Headern vorher
    header('Pragma: cache');
    header('Cache-Control: public');
    header('Expires: ' . date('D, j M Y', strtotime('+1 week')) . ' 00:00:00 GMT');

Bei meiner Lösung nur mit echo funktioniert es komischerweise und ich bekomme 
200 Ok und -> from disk cache

Reply:
sendContent und sendPage zugelich zu benutzen ist nicht richtig. entweder sendet man content oder eine page, aber nicht beides.

Reply:
Natürlich habe ich jeweils nur eines von beiden ausprobiert :)

Reply:
problem war hier vermutlich https://github.com/redaxo/redaxo/pull/1914